l2geth/core/vm/evm.go

```diff
@@ -32,10 +34,66 @@ import (
 	"github.com/ethereum/go-ethereum/rollup/dump"
 )
 
+// codec is a decoder for the return values of the execution manager. It decodes
+// (bool, bytes) from the bytes that are returned from both
+// `ExecutionManager.run()` and `ExecutionManager.simulateMessage()`
+var codec abi.ABI
+
+// innerData represents the results returned from the ExecutionManager
+// that are wrapped in `bytes`
+type innerData struct {
+	Success    bool   `abi:"_success"`
+	ReturnData []byte `abi:"_returndata"`
+}
+
+// runReturnData represents the actual return data of the ExecutionManager.
+// It wraps (bool, bytes) in an ABI encoded bytes
+type runReturnData struct {
+	ReturnData []byte `abi:"_returndata"`
+}
+
 // Will be removed when we update EM to return data in `run`.
 var deadPrefix, fortyTwoPrefix, zeroPrefix []byte
 
 func init() {
+	const abidata = `
```

Comment:
This should be defined as an interface in the contracts repo and I should be able to import its ABI

Reply:
I think that should be possible if we can add it to the spec